bom/pom.xml

```diff
@@ -739,6 +743,19 @@
             </plugin>
          </plugins>
       </pluginManagement>
+      <plugins>
+         <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <version>1.6.8</version>
+            <inherited>false</inherited>
+            <configuration>
+               <nexusUrl>${jboss.releases.nexus.url}</nexusUrl>
+               <serverId>${jboss.releases.repo.id}</serverId>
+               <stagingProfileId>2161b7b8da0080</stagingProfileId>
```

Comment:
Should this be hardcoded and not `${stagingRepositoryId}` instead?

Reply:
I have externalized it and I'm now passing it as a parameter from the Jenkinsfile-release file